### DISCUSSION TITLE
When will the next release be? (Looking for a release without the Unidecode dependency)

### DISCUSSION THREAD

**Discussion by bluemarble** [mindee/doctr#1533] [General]
In my own project, I just noted that the Unidecode package (licensed under GPL) is being brought in as a transitive dependency of python-doctr.  For various reasons, I'm unable to include GPL code in my project.

A quick search here revealed that the maintainers have addressed this already - which is awesome, thanks!  Just wondering when the next release will be out.

**Comment by felixdittrich92** [comment]
Hi @bluemarble 👋,
We have some stuff to solve before we are able to release 0.9.0 
The last release is ~3 weeks ago and we are only two people who work active (in spare time) on docTR so i think ~july/august we are ready for a new release :)
CC @odulcy-mindee

**Reply by odulcy-mindee** [reply]
Yeah, it's a good estimation 👍

**Reply by bluemarble** [reply]
Hi @felixdittrich92, @odulcy-mindee,
I appreciate the quick response, and understand that it may be some time before the release. Thanks!
